client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -165,11 +168,16 @@ function HelpSearchResults( {
 					<div className="inline-help__results-cell">
 						<a
 							href={ localizeUrl( link ) }
-							onClick={ ( event ) => {
-								event.preventDefault();
-								selectSearchResult( resultIndex );
-								onLinkClickHandler( event, result );
-							} }
+							onClick={
+								external
+									? null
+									: ( event ) => {
+											event.preventDefault();
+											selectSearchResult( resultIndex );
+											onLinkClickHandler( event, result );
+									  }
+							}
+							target={ external ? '_blank' : '_self' }
```

Comment:
This line causes an eslint error:
> Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener

I also think it would be better to leave `target` unspecified when `external === false`. Maybe the user has used a browser setting or some extension to open all links in a new tab by default, and when `external === false` we don't really have a preference.

It's a bit messy, but this snippet fixes the lint warning and leaves `target` unspecified by default.
```suggestion
							{ ...( external && {
								target: '_blank',
								rel: 'noreferrer',
							} ) }
```